Psychtoolbox/PsychHardware/EyelinkToolbox/EyelinkBasic/EyelinkBasicLegacy/EyelinkTargetModeDisplay.m

```diff
@@ -0,0 +1,4 @@
+function result=EyelinkTargetModeDisplay(el)
+    warning('EyelinkToolbox:LegacyTargetModeDisplay',['The function EyelinkTargetModeDisplay() is deprecated. Please update your ', ...
```

Comment:
Where's the code to actually implement the legacy target mode display?

Reply:
resolved in 33984c0ef6fc9d2a3dc40f30ab9710ad031d9938